jbmc/src/java_bytecode/java_types.cpp

```diff
@@ -182,6 +182,24 @@ bool is_multidim_java_array_type(const typet &type)
                                        to_struct_tag_type(type.subtype())));
 }
 
+/// Returns the underlying element type and array dimensionality of Java struct
+/// \p type. If this is not an array type the return value will be `{type, 0}`.
+std::pair<typet, std::size_t>
+java_array_dimension_and_element_type(const struct_tag_typet &type)
```

Comment:
Add into the commit message that this is bumping java-models-library, or even better move the bump into a separate commit.